Setup/UpgradeData.php

```diff
@@ -306,4 +308,37 @@ private function setKarCaptureMode($connection)
             );
         }
     }
+    /**
+     * Sets shoppercountry configuration by checking the value of payment/adyen_hpp/country_code
+     *
+     * @param Magento\Framework\DB\Adapter\Pdo\Mysql $connection
+     */
+    private function setShopperCountry($connection)
+    {
+        $countryCode = "payment/adyen_hpp/country_code";
+        $shoppercountry = "payment/adyen_hpp/shoppercountry";
+
+        $select = $connection->select()
+            ->from($this->configDataTable)
+            ->where('path = ?', $countryCode)
+            ->where('value <> "" AND value IS NOT NULL');
+        $configCountryCodeValues = $connection->fetchAll($select);
+
+        foreach ($configCountryCodeValues as $configCountryCodeValue) {
+            $scope = $configCountryCodeValue['scope'];
+            $scopeId = $configCountryCodeValue['scope_id'];
+
+            if (isset($configCountryCodeValue)) {
+                $shoppercountry = 1;
+            }
+            $this->configWriter->save(
+                $shoppercountry,
+                'store_level',
```

Comment:
```suggestion
                1,
```

Reply:
You want to make shoppercountry=1 for every row that is iterated because country_code is not empty nor NULL